kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/container/orchestration/ContainerConfiguration.java

```diff
@@ -163,21 +146,52 @@ public String getContainerLoggingType() {
     }
 
     /**
+     * @since 2.4 Returns the {@link ImageConfiguration} object
+     *
+     * @return
+     */
+    public ImageConfiguration getImageConfiguration() {
+        return this.imageConfig;
+    }
+    
+    /**
+     * @deprecated Since 2.4 use {@link #getImageConfiguration()}. Image related information has been moved to the {@link ImageConfiguration} class.
```

Comment:
Let's follow the pattern linked to deprecate things. Usually we add the deprecation at the end